src/store/cff.ts

```diff
@@ -49,6 +49,25 @@ export function useCff () {
         setTitle: (newTitle: string) => { cff.value.title = newTitle },
         setType: (newType: TypeType) => { cff.value.type = newType },
         setUrl: (newUrl: string) => { cff.value.url = newUrl },
-        setVersion: (newVersion: string) => { cff.value.version = newVersion }
+        setVersion: (newVersion: string) => { cff.value.version = newVersion },
+        reset: () => {
```

Comment:
Not sure that `reset` should stay here but lets leave that another time

Reply:
I needed it for my unit tests and it will be useful when we implement the start another button on finish page.